### ISSUE TITLE
[Bug] CSS Changes are not detected on Build for MAUI Blazor app on Windows

### ISSUE THREAD

**Issue by igalfsg** [dotnet/maui#1729]
### Description
If you change something in the app.css and then try to debug the application, the changes will not be detected until you manually rebuild the application.
### Steps to Reproduce

1. Create a .net MAUI Blazor App
2. Change something in the app.css
3. Run the app in debug mode

### Expected Behavior
the css changes will be detected and the application will change how it looks
### Actual Behavior
nothing changes 
### Basic Information

- Version with issue: preview 6
- Last known good version: N/A
- IDE:Microsoft Visual Studio Community 2019 Preview
Version 16.11.0 Preview 3.0

### Workaround
Manually rebuild the solution when you make a css change.

**Comment by pjmagee** [comment]
Does the BlazorApp become responsive to the Main Window size changing for you? Ive had issues getting that working.

**Comment by igalfsg** [comment]
@pjmagee Yeah I was having that problem too, did you open an issue for it?

**Comment by pjmagee** [comment]
> @pjmagee Yeah I was having that problem too, did you open an issue for it?

No i did not. I dont feel confident in opening up some issues ive come across because im not sure if its a me problem  😂

**Comment by igalfsg** [comment]
2 people having the same problem, I am sure this one is not an us problem, I will open one up :)

**Comment by igalfsg** [comment]
@pjcollins just posted the new issue https://github.com/dotnet/maui/issues/1730 I am not by my dev computer, do you have screenshots you can post in it?

**Comment by pjmagee** [comment]
> @pjmagee just posted the new issue #1730 I am not by my dev computer, do you have screenshots you can post in it?

The default blazor weather app which builds fresh with .NET 6 and was recently updated is having the same problem. So I linked back to that. As its one of the main applications they use for demoing

**Comment by pjmagee** [comment]
> > @pjmagee just posted the new issue #1730 I am not by my dev computer, do you have screenshots you can post in it?
> 
> The default blazor weather app which builds fresh with .NET 6 and was recently updated is having the same problem. So I linked back to that. As its one of the main applications they use for demoing

I think my memory was wrong and they didnt actually show working responsive native MAUI Blazor Control. I think it needs to somehow update its width and height to notify the BlazorControl it has also changed.

**Comment by Eilon** [comment]
I think we've heard a few times about some changes, such as CSS, not getting automatically picked up when re-running a MAUI+Blazor app. We're completely re-doing how CSS files are included in .NET MAUI + Blazor apps, so I'll test this out once those changes are in and see if it fixes the issue.

**Comment by pjmagee** [comment]
> I think we've heard a few times about some changes, such as CSS, not getting automatically picked up when re-running a MAUI+Blazor app. We're completely re-doing how CSS files are included in .NET MAUI + Blazor apps, so I'll test this out once those changes are in and see if it fixes the issue.

It picks up everything. The problem is, the Window is not telling the WebView that it changed in size...or something like this. So the CSS might be fine. But the Blazor WebView doesnt know its parent window is resized and so the content does not change, which is i guess a bit weird if your main app in the Window/Maui Window is a full Blazor WebView control. I'm not sure how Maui Blazor controls should react if they arent the main full Control of the Page/Window.

**Comment by pjmagee** [comment]
> > I think we've heard a few times about some changes, such as CSS, not getting automatically picked up when re-running a MAUI+Blazor app. We're completely re-doing how CSS files are included in .NET MAUI + Blazor apps, so I'll test this out once those changes are in and see if it fixes the issue.
> 
> It picks up everything. The problem is, the Window is not telling the WebView that it changed in size...or something like this. So the CSS might be fine. But the Blazor WebView doesnt know its parent window is resized and so the content does not change, which is i guess a bit weird if your main app in the Window/Maui Window is a full Blazor WebView control. I'm not sure how Maui Blazor controls should react if they arent the main full Control of the Page/Window. But if people are using Maui Blazor with the intention that its basically a full size blazor app wrapped in a Maui Page wrapped in a Native WinUI3 Window, somehow the Window Size Change events arent bubbling down the chain.

Sorry, @Eilon  Ignore me - This issue was specific for CSS changes (Not the Responsive thing) Excuse my comment I got mixed up since we raised a seperate issue for that.

**Comment by Eilon** [comment]
No worries, I figured as much 😄

**Comment by Eilon** [comment]
Hi everyone, as far as I know all these issues are fixed for the next preview. I used to see such issues myself a while ago but not recently in the `main` branch of this repo, either in the dev solution, or using the project templates that I built locally. As such, I am closing this issue. But if someone sees any such issue after the next preview (should be this month), I would love to hear about it and make sure we address any remaining issues.
